.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -97,16 +41,16 @@ body:
     id: operating-system-version
     attributes:
       label: Which operating system version?
-      placeholder: e.g. Debian 11
+      placeholder: Debian 11 (Bullseye)
     validations:
       required: true
   - type: dropdown
     id: isa
     attributes:
-      label: What instruction set architecture is DMS running on?
+      label: What computer architecture is the host running?
```

Comment:
Assuming this change is because ISA won't make sense to many? :sweat_smile: (_the choices should regardless though_)